nixos/modules/services/security/certmgr.nix

```diff
@@ -0,0 +1,210 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.certmgr;
+
+  allSpecs = pkgs.buildEnv {
+    name = "certmgr.d";
+    paths = mapAttrsToList (name: value: pkgs.writeTextDir (name + ".json") (builtins.toJSON value)) cfg.declSpecs
+         ++ optional (cfg.impSpecs != []) (pkgs.linkFarm "impSpecs" cfg.impSpecs);
+  };
```

Comment:
Changed as part of https://github.com/NixOS/nixpkgs/pull/44556/commits/e77155cbf3eafde44bc923e6f19b0e974a86c8f2 .. Still need to distinguish specs of type attrset and specs of type path, since the former needs to be written to the store before linking.